src/test/java/org/passay/AllowedCharacterRuleTest.java

```diff
@@ -75,6 +75,33 @@ public Object[][] passwords()
           new PasswordData("gbwersco4kk"),
           null,
         },
+        // test unicode
+        {
+          new AllowedCharacterRule(ALLOWED_CHARS),
```

Comment:
I had hoped to see some test coverage indicating how to create rules using code points. That's the heart of the change, and seeing at least one test that demonstrates it seems warranted.